src/Admin/Views/Users/Edit.cshtml

```diff
@@ -126,39 +152,52 @@
             <div class="form-group">
                 <label asp-for="GatewayCustomerId"></label>
                 <div class="input-group">
-                    <input type="text" class="form-control" asp-for="GatewayCustomerId">
-                    <div class="input-group-append">
-                        <button class="btn btn-secondary" type="button" id="gateway-customer-link">
-                            <i class="fa fa-external-link"></i>
-                        </button>
-                    </div>
+                    <input type="text" class="form-control" asp-for="GatewayCustomerId" disabled='@(canEditBilling ? null : "disabled")'>
+                    @if (canLaunchInStripe)
```

Comment:
❓  the gateway is implemented using a select which implies we can have non-stripe billing gateways, but then we name this permission specifically to be `canLaunchInStripe`?   Do we have non-Stripe billing gateways?
![image](https://user-images.githubusercontent.com/116684653/223256269-a44a03fb-fa57-4494-960b-05a99e7a86e4.png)

Reply:
Great point. We also have Braintree. The requirements specifically called out launching in Stripe, so I'll confirm if the permission check should be applied to just Stripe or to all gateways, and either update the name of the permission or the logic around it depending on the answer.

Reply:
Awesome. Thank you!

Reply:
@dgoodman-bw do you want me to go ahead and approve this or wait on this potential refactoring?

Reply:
@JaredSnider-Bitwarden Turns out the permission is intended to include all billing gateways. The change has been implemented and is now ready for re-review. Thank you!